gtsfm/configs/sift_front_end_astrovision.yaml

```diff
@@ -1,7 +1,7 @@
 SceneOptimizer:
   _target_: gtsfm.scene_optimizer.SceneOptimizer
   save_gtsfm_data: True
-  save_two_view_correspondences_viz: False
+  save_two_view_correspondences_viz: True
```

Comment:
this will blow up the artifact size i believe -- is it needed?

Reply:
fixed